docs/07-wsl-setup.md

```diff
@@ -0,0 +1,51 @@
+# How to run the webapp on linux subsystem on Windows (WSL1)?
+Most of the webapp as been developed from a Linux environment.
+To avoid environment specific troubleshooting and to improve performance it is recommended to set up the Windows Subsystem fot Linux (WSL) on Windows.
+
+## Setup
+
+1. Enable the linux subsystem ([Official Guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10))
+- In the powershell run:
+```powershell
+dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+```
+- Restart Windows
+- Install a Linux distribution of your choice (see Microsoft Store)
```

Comment:
The following seems to be Ubuntu specific. If that is the case, probably add a note mentioning that.